### PR TITLE
add how to monitor a job from aml studio.

### PR DESCRIPTION
**Pull request by acured**

### CODE REVIEW

File: docs/en_US/TrainingService/AMLMode.rst [hunk 1]
```diff
@@ -82,6 +82,37 @@ Compared with `LocalMode <LocalMode.rst>`__ trial configuration in aml mode have
 
   * required key. The docker image name used in job. NNI support image ``msranni/nni`` for running aml jobs.
 
+Monitor your code in the cloud by using the studio
+--------------------------------------------------
+
+To monitor your job's code, you need visit your studio which you create at step 5. Once the job completes, go to the Outputs + logs tab. There you can see a 70_driver_log.txt file that looks like this:
+
+.. code-block:: yaml
```
Comment (ultmaster) [inline docs/en_US/TrainingService/AMLMode.rst:90]: Why is it a yaml?

Comment (acured) [reply docs/en_US/TrainingService/AMLMode.rst]: Should be 'text', changed.

File: docs/en_US/TrainingService/AMLMode.rst [hunk 2]
```diff
@@ -82,6 +82,11 @@ Compared with `LocalMode <LocalMode.rst>`__ trial configuration in aml mode have
 
   * required key. The docker image name used in job. NNI support image ``msranni/nni`` for running aml jobs.
 
+Monitor your code in the cloud by using the studio
+--------------------------------------------------
+
+To monitor your job's code, you need visit your studio which you create at step 5. Once the job completes, go to the Outputs + logs tab. There you can see a 70_driver_log.txt file, This file contains the standard output from a run and can be useful when you're debugging remote runs in the cloud. Learn more about aml from `here <https://docs.microsoft.com/en-us/azure/machine-learning/tutorial-1st-experiment-hello-world>`__.
+
```
Comment (J-shang) [inline docs/en_US/TrainingService/AMLMode.rst:89]: please place this part at the end of this doc, the following config reference also belongs to the section `Run an experiment`.

you need visit -> you need to visit

Comment (acured) [reply docs/en_US/TrainingService/AMLMode.rst]: Done.

**Review by J-shang — APPROVED** [review]

File: docs/en_US/TrainingService/AMLMode.rst [hunk 3]
```diff
@@ -125,3 +125,8 @@ Run the following commands to start the example experiment:
    nnictl create --config config_aml.yml
 
 Replace ``${NNI_VERSION}`` with a released version name or branch name, e.g., ``v2.0``.
+
+Monitor your code in the cloud by using the studio
```
Comment (QuanluZhang) [inline docs/en_US/TrainingService/AMLMode.rst:129]: "cloud" and "studio" are the words used in AML?

Comment (acured) [reply docs/en_US/TrainingService/AMLMode.rst:129]: Yes.  It comes from [here](Words used in AML docs.  It comes from [here](https://docs.microsoft.com/en-us/azure/machine-learning/tutorial-1st-experiment-hello-world#monitor)

Comment (QuanluZhang) [reply docs/en_US/TrainingService/AMLMode.rst:129]: got it, thanks

**Review by QuanluZhang — APPROVED** [review]

**Pull request merged**